README.md

```diff
@@ -12,19 +12,43 @@ Set the following values in your Shopgate Connect Admin:
 * `smartCategories` - (array of objects) each containing the following properties
   * `id` - (text/number, required) Unique ID for the category.
   * `name` - (text, required) Name of the category that will be shown to the end user.
-  * `filters` - (object, required) The filters in CloudSearch syntax.
+  * `searchPhrase` - (text, optional) A searchPhrase to be executed when the user opens this category.
+  * `filters` - (object, optional) The filters in CloudSearch syntax.
+  * `productIds` - (array, optional) List of product IDs.
   * `description` - (text, optional) Not diplayed anywhere atm. Just for documentary purposes.
   * `imageUrl` - (text, optional) Link to the category image.
-  * `externalUrl` - (text, optional)  Link to the category in the desktop shop.
+  * `externalUrl` - (text, optional) Link to the category in the desktop shop.
+  * `hideFromTree` - (true/false, optional, default: false) Don't show the smart category in the category tree.
```

Comment:
Please reverse the config to say "showInCategoryTree" and that it defaults to true.